Applications/spire/source/book_view_ui_tester/random_book_view_model.cpp

```diff
@@ -93,6 +93,16 @@ QtPromise<void> RandomBookViewModel::load() {
         for(auto i = 0; i < 1000; ++i) {
           update();
         }
+        Quote bid_quote(Money(100), 100, Side::BID);
+        Quote ask_quote(Money(100), 100, Side::ASK);
+        for(auto i = 0; i < 100; ++i) {
+          bid_quote.m_price = (100 * Money::ONE) - (i * Money::CENT);
+          m_bids.push_back(BookQuote("TST", true, DefaultMarkets::TSX(),
+            bid_quote, ptime()));
```

Comment:
For the timestamp, use the current universal time. Refer to boost documentation to see how to get that value.